### PR TITLE
Update install-app-guard.md

### PR DESCRIPTION
**Pull request by sazankha**
Updated minimum system requirements

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ information about system requirements and installation steps to enable Applicati
 ### Minimum software requirements
 
 * **Windows 10**: Windows 10 Enterprise edition, Client Build version 2004 (20H1) build 19041 or later
-* **Office**: Office Current Channel Build version 2011 16.0.13530.10000 or later
+* **Office**: Office Current Channel Build version 2011 16.0.13530.10000 or later. Both 32-bit and 64-bit versions of office are supported.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/install-app-guard.md]:
```suggestion
* **Office**: Office Current Channel Build version 2011 16.0.13530.10000 or later. Both 32-bit and 64-bit versions of Office are supported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@schmurky Please merge this PR. Thanks

**Pull request merged**